backend/Makefile

```diff
@@ -15,8 +15,11 @@ backend:
 .PHONY: backend
 
 run:
+	DB_URL?= $(az cosmosdb show -n ${DB_NAME} -g ${RESOURCEGROUP} --query documentEndpoint -o tsv) && \
```

Comment:
Is the ? A valid symbol here?

Reply:
No, it should be the same as the deployment.